components/decentralized_dns/decentralized_dns_opt_in_page.cc

```diff
@@ -69,13 +69,14 @@ void DecentralizedDnsOptInPage::PopulateInterstitialStrings(
     base::Value* load_time_data) {
   const std::vector<std::u16string> message_params = {
       u"<a "
-      u"href='https://www.cloudflare.com/en-ca/"
-      u"distributed-web-gateway-terms/' target='_blank' "
-      u"rel='noopener noreferrer'>",
+      u"href='https://consensys.net/terms-of-use/' "
```

Comment:
I believe because this is changing service providers we should get at least a Privacy Review of the change of services.

Reply:
cc @privacymatters - probably worth opening a privacy review issue unless he's able to quickly review this here